orbit/README.md

```diff
@@ -152,7 +152,7 @@ Configure update channels for Orbit and osqueryd with the `--orbit-channel` and
 | `4.6`   | 4.6.x    |
 | `4.6.0` | 4.6.0    |
 
-Additionally `stable` and `edge` are special channel names. `stable` will always return the version Fleet deems to be stable, while `edge` will provide newer releases for beta testing.
+Additionally `stable` and `edge` are special channel names. `stable` will always return the version Fleet deems to be stable. When a new version of osquery is released, it is added to the `edge` channel for beta testing. Fleet then provides input to the osquery TSC based on testing. After the version is declared stable by the osquery TSC, Fleet will promote the version to `stable` ASAP.
```

Comment:
```suggestion
Additionally `stable` and `edge` are special channel names. `Stable` will always return the version Fleet deems to be stable. When a new version of osquery is released, it is added to the `edge` channel for beta testing. Fleet then provides input to the osquery TSC based on testing. After the version is declared stable by the osquery TSC, Fleet will promote the version to `stable` ASAP.
```
I'm not sure if this should stay lowercase like "macOS" when used at the beginning of a sentence. I'll defer to you if it should be changed or stay as is.

Reply:
I think it should stay lowercase, but I agree that it looks funky. The simplest solution is just to not put it at the beginning of a sentence. Commit incoming!